src/bpm/runc/adapter/adapter.go

```diff
@@ -29,10 +29,27 @@ import (
 )
 
 const (
-	ResolvConfDir string = "/run/resolvconf"
-	DefaultLang   string = "en_US.UTF-8"
+	resolvConfDir string = "/run/resolvconf"
+	defaultLang   string = "en_US.UTF-8"
 )
 
+var defaultPrivilegedCapabilities = []string{
```

Comment:
may want to link to that docker place you got this from.

Reply:
+1 link to docs or code would be great